backend/app/services/knowledge/knowledge_service.py

```
@@ -257,18 +258,48 @@ def list_knowledge_bases(
+            shared_kb_ids = [p.resource_id for p in shared_permissions]
+
+            # Single query to get both personal and shared knowledge bases
+            # Personal: user_id matches and namespace is "default"
+            # Shared: id is in shared_kb_ids
+            all_kbs = (
                 db.query(Kind)
                 .filter(
                     Kind.kind == "KnowledgeBase",
-                    Kind.user_id == user_id,
-                    Kind.namespace == "default",
                     Kind.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)